pkgs/tools/system/acpica-tools/default.nix

```
@@ -31,7 +31,8 @@ stdenv.mkDerivation rec {
 
   NIX_CFLAGS_COMPILE = [
     "-O3"
-    # Needed with GCC 12
+  ] ++ lib.optional stdenv.cc.isGNU [
```

```suggestion
  ] ++ lib.optionals stdenv.cc.isGNU [
```

Also, please rebase.